cli/coredump/coredump.go

```diff
@@ -0,0 +1,212 @@
+package coredump
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"embed"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/apex/log"
+)
+
+//go:embed scripts/*
+var corescripts embed.FS
+
+// readEmbedFile reads content of embed file.
+func readEmbedFile(fs embed.FS, path string) (string, error) {
```

Comment:
Seems like common function. If so it can be moved to `util.go`.

Reply:
Fixed